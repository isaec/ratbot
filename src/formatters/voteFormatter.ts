import { SlackBlock } from "@src/helpers/slack/blocks";

export type UserId = string;
export type VoteData = {
  for: UserId[];
  against: UserId[];
};

export const parseExistingVote = (existingVote: SlackBlock): VoteData => {
  return null;
};

export const formatVote = (voteData: VoteData): SlackBlock => {
  return null;
};
