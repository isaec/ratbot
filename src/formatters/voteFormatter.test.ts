import { generate, some } from "generate-combinations";
import { describe, expect, test } from "vitest";
import { formatVote, VoteData } from "./voteFormatter";

const voteData = generate<VoteData>({
  for: some(["ian", "andy", "mark", "taylor"]),
  against: some(["jasmine", "isaac", "zach"]),
});

describe("formatVote", () => {
  test.each(voteData)("formatVote(%o)", (voteData) => {
    expect(formatVote(voteData)).toMatchSnapshot();
  });
});
