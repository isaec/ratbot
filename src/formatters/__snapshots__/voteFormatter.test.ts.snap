// Vitest Snapshot v1

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', 'taylor', [length]: 3 ],
  against: [ [length]: 0 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'mark', [length]: 2 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', 'taylor', [length]: 2 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', [length]: 1 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', [length]: 1 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', [length]: 1 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'andy', [length]: 1 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', 'taylor', [length]: 4 ],
  against: [ [length]: 0 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'mark', [length]: 3 ],
  against: [ [length]: 0 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', 'taylor', [length]: 3 ],
  against: [ [length]: 0 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'andy', [length]: 2 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', 'taylor', [length]: 3 ],
  against: [ [length]: 0 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'mark', [length]: 2 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', 'taylor', [length]: 2 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', [length]: 1 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', [length]: 1 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', [length]: 1 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'ian', [length]: 1 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'isaac', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'jasmine', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', 'taylor', [length]: 2 ],
  against: [ 'zach', [length]: 1 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', [length]: 1 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', [length]: 1 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', [length]: 1 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'mark', [length]: 1 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'taylor', [length]: 1 ],
  against: [ 'isaac', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'taylor', [length]: 1 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'taylor', [length]: 1 ],
  against: [ 'jasmine', 'isaac', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ 'taylor', [length]: 1 ],
  against: [ 'jasmine', 'zach', [length]: 2 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({
  for: [ [length]: 0 ],
  against: [ 'jasmine', 'isaac', 'zach', [length]: 3 ]
}) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'andy', 'mark', [length]: 2 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'andy', 'taylor', [length]: 2 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'andy', [length]: 1 ], against: [ 'isaac', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'andy', [length]: 1 ], against: [ 'jasmine', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'andy', [length]: 1 ], against: [ 'zach', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'andy', [length]: 1 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', 'andy', [length]: 2 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', 'mark', [length]: 2 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', 'taylor', [length]: 2 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', [length]: 1 ], against: [ 'isaac', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', [length]: 1 ], against: [ 'jasmine', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', [length]: 1 ], against: [ 'zach', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'ian', [length]: 1 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'mark', 'taylor', [length]: 2 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'mark', [length]: 1 ], against: [ 'isaac', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'mark', [length]: 1 ], against: [ 'jasmine', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'mark', [length]: 1 ], against: [ 'zach', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'mark', [length]: 1 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'taylor', [length]: 1 ], against: [ 'isaac', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'taylor', [length]: 1 ], against: [ 'jasmine', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'taylor', [length]: 1 ], against: [ 'zach', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ 'taylor', [length]: 1 ], against: [ [length]: 0 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ 'isaac', 'zach', [length]: 2 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ 'isaac', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ 'jasmine', 'isaac', [length]: 2 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ 'jasmine', 'zach', [length]: 2 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ 'jasmine', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ 'zach', [length]: 1 ] }) 1`] = `null`;

exports[`formatVote > formatVote({ for: [ [length]: 0 ], against: [ [length]: 0 ] }) 1`] = `null`;
